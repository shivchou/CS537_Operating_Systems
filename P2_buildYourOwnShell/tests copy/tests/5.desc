External command execution with path manipulation
