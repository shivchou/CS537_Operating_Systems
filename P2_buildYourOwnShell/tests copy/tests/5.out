/bin
tmp:/bin
Dummy Echo
/bin:tmp
2

